### PR TITLE
feat(@angular-devkit/build-karma): add package

### PR DESCRIPTION
**Pull request by filipesilva**

### CODE REVIEW

File: debug.log [hunk 1]
```diff
@@ -0,0 +1 @@
+[0604/171207.114:ERROR:prune_crash_reports.cc(34)] PruneCrashReportDatabase: Failed to get pending reports
```
Comment (filipesilva) [inline debug.log]: This file shouldn't be here.

File: packages/angular_devkit/build_karma/README.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# Karma Builder for Architect
+
+WIP
```
Comment (filipesilva) [inline packages/angular_devkit/build_karma/README.md]: Needs to be filled in.

**Comment by filipesilva** [comment]
It also looks like the test runner is not exiting. This is likely because of the karma watch tests, since karma doesn't have a good way to exit gracefully when watching.

**Comment by alexeagle** [comment]
Sorry @filipesilva, this got lost when we moved the code back over to the angular/angular-cli repo. Do you have time to re-base this commit on master over there?

**Pull request closed without merging**